### PR TITLE
Chore zero and decimals for bei tei pei total life time

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ZH CARD 1437](https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1437)

### CODE REVIEW

File: app/lib/theme/widgets/ReadOnlyNumberWidget.tsx [hunk 1]
```diff
@@ -18,24 +19,32 @@ const ReadOnlyNumberWidget: React.FC<WidgetProps> = ({
   return (
     <dd>
       {value ? (
```
Comment (dleard) [inline app/lib/theme/widgets/ReadOnlyNumberWidget.tsx:21]: This `value ? ` conditional is causing the summary (and any other readOnly view) to show 'Not Added' for values of 0

**Review by dleard — CHANGES_REQUESTED** [review]
Looks good for the most part. Just a bug in the ReadOnlyNumberWidget where 0 values are being displayed as 'Not Added'. Once that is fixed, some tests should be added to the summary test file for Emission Intensity Report to ensure the readonly values are displaying as expected.

File: app/lib/theme/widgets/ReadOnlyNumberWidget.tsx (hunk 1 above)
Comment (Sepehr-Sobhani) [reply app/lib/theme/widgets/ReadOnlyNumberWidget.tsx:21]: @dleard Thanks for catching this 🙏 This led to a bigger problem with saving zero value and our diffing.

**Review by dleard — APPROVED** [review]
:100: awesome work. I checked that zeroes and decimals can be set for the EI form fields, that the zero values are persisted to the database table on submit and that the read-only summary page shows the 0 values properly

**Pull request merged**